samples/ImplicitFlow01/AppWithFetchClient/wwwroot/app.js

```diff
@@ -0,0 +1,72 @@
+/* global window, fetch, Oidc */
+
+//
+// Define helper functions.
+//
+const fetchUserResources = (accessToken) => {
+  const url = `${window.location.origin}/api`;
+  const options = {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  };
+
+  // See https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+  fetch(url, options)
+    .then(response => response.json())
+    .then(obj => JSON.stringify(obj))
```

Comment:
Consistent formatting
https://github.com/openiddict/openiddict-samples/pull/68/files#diff-51cc0f9258512075b52c1e08f08906d6R26

Reply:
I think I resolved that too, though I wasn't entirely sure what you meant.